### DISCUSSION TITLE
mqtt cluster

### DISCUSSION THREAD

**Discussion by attiliodrei** [processone/ejabberd#3601] [Questions] [answered]
in a ejabberd 3 node mqtt cluster  last docker ejabberd/ecs
if on the first node there  are not attached subscibers,  and publish done on first node client connected to other nodes does not receive messages

**Comment by badlop** [comment, marked as answer]
Right! This problem was reported again in https://github.com/processone/ejabberd/issues/3750 and I finally could investigate it. The solution is being tested nowadays, and will be committed to ejabberd when it's polished enough.
